1.8/tutorial/service-discovery.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: Connecting Apps/Service Discovery
+nav_title: Service Discovery
+menu_order: 4
+---
+
+# Prerequisites
+In order to get started with this tutorial, you should have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, we expect you have just deployed app1 in the previous step of this tutorial.
+
+# Objective
+Our application in the previous part used`redis.marathon.l4lb.thisdcos.directory:6379` as the address for connecting to redis. As redis might be running on any agent in the cluster (and furthermore on different ports), how does this address link to the actual running redis instance?
+In this section, we will learn about DC/OS service discovery by exploring the different options for service discovery for apps in DC/OS.
+
+# Steps
+  [Service discovery](https://dcos.io/docs/1.8/usage/service-discovery/)  is very important as we actually want to be ignorant (at least most of the time) about where our apps are running, but still be able to connect to them. This is especially true in cases where apps fail and might be restarted on a different host.
```

Comment:
Suggest: `Service discovery(link) allows you to connect to your applications without necessarily knowing where they are running. Service discovery is particularly useful in cases where applications may fail and be restarted on a different host.`